src/web/services/log.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { LogTypes, ResourceEndpoints } from '../types/api-const';
+import { HttpRequestService } from './http-request.service';
+
+/**
+ * Handles logging related logic provision.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class LogService {
+
+  constructor(private httpRequestService: HttpRequestService) { }
+
+  /**
+   * Creates a log for feedback session by calling API.
+   */
+  createFeedbackSessionLog(queryParams: {
+    courseId: string,
+    studentEmail: string,
+    logType: LogTypes }): Observable<string> {
+    const paramMap: Record<string, string> = {
+      courseId: queryParams.courseId,
+      studentEmail: queryParams.studentEmail,
```

Comment:
`courseid` and `studentemail` should be in lowercase here

Reply:
Fixed. Thanks